.github/actions/install_build_deps/action.yml

```diff
@@ -0,0 +1,49 @@
+name: "Install Wazuh build dependencies"
+description: "This action installs all the requirements to test and build Wazuh agent/server in Ubuntu Linux"
+inputs:
+   target:
+     required: true
+     description: "Wazuh target to decide if install or not custom dependencies"
+runs:
+  using: "composite"
+  steps:
+    - name: Install tools and libraries
+      shell: bash
+      run: sudo apt install cppcheck astyle valgrind lcov clang-tools libcmocka-dev -y
```

Comment:
In case of compilation for "winagent" target, libcmocka is build from sources.
So, in this step is not necessary to install "libcmocka-dev" package when target is "winagent".

Reply:
It's true.
I'll install the library only once according to the target.